package.json

```diff
@@ -54,36 +52,36 @@
     "zustand": "4.1.4"
   },
   "devDependencies": {
-    "@testing-library/jest-dom": "^5.14.1",
-    "@testing-library/react": "^12.1.1",
-    "@testing-library/user-event": "^13.2.1",
+    "@testing-library/jest-dom": "^5.17.0",
+    "@testing-library/react": "^14.0.0",
+    "@testing-library/user-event": "^14.4.3",
     "@types/jest": "^27.0.2",
     "@types/react": "^18.0.0",
     "@types/react-dom": "^18.0.0",
     "@types/styled-components": "^5.1.14",
     "@types/three": "0.152.1",
     "@types/uuid": "^8.3.1",
-    "@typescript-eslint/eslint-plugin": "^4.32.0",
-    "@typescript-eslint/parser": "^4.32.0",
-    "babel-plugin-import": "^1.13.3",
-    "customize-cra": "^1.0.0",
-    "customize-cra-react-refresh": "^1.1.0",
-    "eslint": "^7.32.0",
-    "eslint-config-prettier": "^8.3.0",
-    "eslint-plugin-promise": "^5.1.0",
-    "eslint-plugin-react": "^7.26.1",
-    "eslint-plugin-react-hooks": "^4.2.0",
+    "@typescript-eslint/eslint-plugin": "^6.2.1",
+    "@typescript-eslint/parser": "^6.2.1",
+    "@vitejs/plugin-react": "^4.0.4",
+    "babel-plugin-import": "^1.13.8",
+    "eslint": "^8.46.0",
+    "eslint-config-prettier": "^8.10.0",
+    "eslint-plugin-no-only-tests": "^3.1.0",
+    "eslint-plugin-prettier": "^5.0.0",
+    "eslint-plugin-promise": "^6.1.1",
+    "eslint-plugin-react": "^7.33.1",
+    "eslint-plugin-react-hooks": "^4.6.0",
     "husky": "^7.0.2",
-    "less": "^3.12.2",
-    "less-loader": "^5.0.0",
+    "less": "^4.1.3",
     "lint-staged": "^11.1.2",
     "prettier": "^2.4.1",
-    "react-app-rewired": "^2.1.8",
-    "react-scripts": "4.0.3",
     "rimraf": "^3.0.2",
-    "ts-loader": "^9.2.6",
-    "typescript": "4.4.3",
-    "url-loader": "^4.1.1",
+    "typescript": "^5.1.6",
```

Comment:
As a note, typescript version (and eslint / testing library versions as well) should also be updated later in buerli / buerli-react-cad / etc.

Reply:
Absolutely agree! Furthermore, we should think about moving to vite for examples and modeler as well.